media-gfx/ufraw/files/ufraw-0.22-jasper-automagic.patch

```diff
@@ -0,0 +1,24 @@
+--- configure.ac
++++ configure.ac
+@@ -158,9 +158,17 @@
+   AC_CHECK_LIB(jpeg, jpeg_CreateCompress))
+ have_jpeg=${ac_cv_lib_jpeg_jpeg_CreateCompress:-no}
+ 
+-# Check for libjasper.
+-AC_CHECK_LIB(jasper, jas_image_decode)
+-have_jasper=${ac_cv_lib_jasper_jas_image_decode:-no}
++AC_ARG_ENABLE([jasper],
++  AC_HELP_STRING([--enable-jasper],[enable JPEG2000 support via jasper]),
++  ,enable_jasper=no)
```

Comment:
leave away the `enable_jasper=no`, that is not how options should be handled https://autotools.io/autoconf/arguments.html Also use `AS_HELP_STRING`